File: Medium/1887. Reduction Operations to Make the Array Elements Equal/1887. Reduction Operations to Make the Array Elements Equal.java
import java.util.Arrays;

class Solution {
    public int reductionOperations(int[] nums) {
        Arrays.sort(nums);
        int operations = 0;
        int nextLargest = nums[nums.length - 1];

        for (int i = nums.length - 2; i >= 0; i--) {
            if (nums[i] < nextLargest) {
                nextLargest = nums[i];
                operations += nums.length - i - 1;
            }
        }

        return operations;
    }
}

File: Medium/1887. Reduction Operations to Make the Array Elements Equal/1887. Reduction Operations to Make the Array Elements Equal.py
from typing import List

class Solution:
    def reductionOperations(self, nums: List[int]) -> int:
        nums.sort(reverse=True)  # Sort in descending order

        operations = 0
        next_largest = nums[0]

        for i in range(1, len(nums)):
            if nums[i] < next_largest:
                next_largest = nums[i]
                operations += i

        return operations
